t/14-grutasks-git.t

```diff
@@ -448,12 +449,108 @@ subtest 'delete_needles' => sub {
     like $error->{message}, qr{Unable to rm via Git}, 'expected error from git';
 
     subtest 'minion guard' => sub {
-        my $guard = $t->app->minion->guard("git_clone_t/data/openqa/share/tests/fedora/needles_task", ONE_HOUR);
+        my $guard = $t->app->minion->guard('git_clone_t/data/openqa/share/tests/fedora/needles_task', ONE_HOUR);
         $res = run_gru_job(@gru_args);
         is $res->{state}, 'finished', 'job finished';
         $error = $res->{result}->{errors}->[0];
         like $error->{message}, qr{Another git task for.*fedora.*is ongoing}, 'expected error message';
     };
+
+};
+
+subtest 'ServerAvailability.pm: no lock file => skip' => sub {
+    my $tmpdir = tempdir();
+    local $ENV{OPENQA_GIT_SERVER_OUTAGE_FILE} = "$tmpdir/mock";
+    my $outcome = OpenQA::Git::ServerAvailability::report_server_unavailable($t->app, 'gitlab');
```

Comment:
Maybe you can export `report_server_unavailable` from `OpenQA::Git::ServerAvailability`?